fiboa_cli/datasets/nl.py

```diff
@@ -0,0 +1,101 @@
+from ..convert_utils import convert as convert_
+
+URI = "https://service.pdok.nl/rvo/referentiepercelen/atom/downloads/referentiepercelen.gpkg"
+
+ID = "nl"
+TITLE = "Field boundaries for The Netherlands"
+DESCRIPTION = """
+A field block (Dutch: "Referentieperceel"), formerly known as "AAN" (Agrarisch Areaal Nederland),
+is a contiguous agricultural area surrounded by permanent boundaries, which is cultivated by one or
+more farmers with one or more crops, is fully or partially set aside or is fully or partially
+taken out of production.
+
+The following field block types exist:
+
+- Woods (Hout)
+- Agricultural area (Landbouwgrond)
+- Other (Overig)
+- Water (Water)
+
+We filter on "Agricultural area" in this converter.
+For crop data, look at BasisRegistratie gewasPercelen (BRP)
+"""
+BBOX = [2.35417303, 50.71447164, 7.5553525, 55.66948102]
+
+PROVIDER_NAME = "RVO / PDOK"
+PROVIDER_URL = "https://www.pdok.nl/introductie/-/article/referentiepercelen"
+ATTRIBUTION = None
+LICENSE = "CC0"  # http://creativecommons.org/publicdomain/mark/1.0/
+
+ADD_COLUMNS = {
+    "determination_datetime": "2023-06-15T00:00:00Z"
+}
+
+EXTENSIONS = []
+COLUMN_MIGRATIONS = []
+COLUMNS = {
+    'geometry': 'geometry',
+    'id': 'id',
+    'area': "area",
+    'versiebron': 'source',
+    'type': 'category'
+}
+
+COLUMN_FILTERS = {
+    # category = "Hout" | "Landbouwgrond" | "Overig" | "Water"
+    "category": lambda col: col == "Landbouwgrond"
+}
+
+
+def migrate(gdf):
+    # Projection is in CRS 28992 (RD New), this is the area calculation method of the source organization
+    gdf['area'] = gdf.area / 10000
+    # index attribute is available through pyogrio
+    gdf['id'] = gdf.index
+    # Convert accidental multipolygon type to polygon
+    gdf = gdf.explode(index_parts=False)
+    return gdf
+
+
+MIGRATION = migrate
+
+MISSING_SCHEMAS = {
+    "properties": {
+        "source": {
+            "type": "string"
+        },
+        "category": {
+            "type": "string"
```

Comment:
Should this be an enum just with `Landbouwgrond` or should this column be completely removed because it's just one value?

Reply:
Let's remove it. If relevant, it could be reintroduced again...